Source/OxyPlot/Series/DataRange.cs

```diff
@@ -0,0 +1,111 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="DataRange.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Represents an interval defined by two doubles
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Series
+{
+    using System;
+
+    /// <summary>
+    /// Represents an interval defined by two doubles.
+    /// </summary>
+    public struct DataRange : ICodeGenerating
+    {
+        /// <summary>
+        /// The undefined data range.
+        /// </summary>
+        public static readonly DataRange Undefined = new DataRange(double.NaN, double.NaN);
+
+        private double? minimum;
+        private double? maximum;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DataRange"/> struct.
+        /// </summary>
+        /// <param name="min">lower bound.</param>
```

Comment:
Should use proper sentence case in param comments, e.g. "The inclusive lower bound."

Reply:
Done.